lumicks/pylake/tests/data/mock_widefield.py

```diff
@@ -146,17 +146,21 @@ def make_alignment_image_data(spots, red_warp_parameters, blue_warp_parameters,
 
     reference_image = np.repeat(green_image[:,:,np.newaxis], 3, axis=2)
     warped_image = np.stack((red_image, green_image, blue_image), axis=2).squeeze()
-    description = make_wt_description(version, bit_depth, m_red, m_blue, offsets)
+    if camera == "wt":
+        description = make_wt_description(version, bit_depth, m_red, m_blue, offsets)
+    elif camera == "irm":
+        description = make_irm_description(version, bit_depth)
+        # by definition, grayscale images only have 1 channel
+        reference_image = reference_image[:,:,1]
+        warped_image = warped_image[:,:,1]
+    else:
+        raise ValueError("camera argument must be 'wt' or 'irm'")
 
     return reference_image, warped_image, description, bit_depth
 
 
 def write_tiff_file(reference_image, warped_image, description, bit_depth, n_frames, filename):
```

Comment:
`bit_depth` is still needed because that field isn't in the `v1` form of the metadata but is needed in the tests for some things.

Also, `reference_image` is used in the tests (for example `test_image_reconstruction_rgb`)

Reply:
I'm talking about `write_tiff_file` specifically. These fields are not used in the function. The only place I see that function used is `test_correlated_stack`. I assume they're there because of the old format and directly being able to unpack the alignment data with `*`. Maybe rename the args to `_` if they're unused to indicate that we know about it.